tests/test_tinypages.py

```diff
@@ -21,6 +23,7 @@
 ENVIRONMENT_HOOKS = ['PLOT_SKIP', 'PLOT_SKIP_OPTIONAL']
 
 
+@flaky_test
```

Comment:
Might need to add `FileExistsError` to the decorator to catch and retry this failure:
https://github.com/pyvista/pyvista/actions/runs/12185900129/job/33993171811?pr=6938#step:14:2212

``` suggestion
@flaky_test(exceptions=(AssertionError, FileExistsError))
```

Reply:
Happened again:
https://github.com/pyvista/pyvista/actions/runs/12208190340/job/34060977087?pr=6946#step:14:2721
will open a PR